liquidctl/driver/kraken3.py

```diff
@@ -311,18 +311,13 @@ def get_status(self, direct_access=False, **kwargs):
         Returns a list of `(property, value, unit)` tuples.
         """
 
-        # no driver currently supports pwm1, so silently fallback to
-        # direct mode if it isn't available; for the same reason, also don't
-        # yet issue a warning when directly accessing the device
-
-        if self._hwmon and not direct_access and self._hwmon.has_attribute("pwm1"):
```

Comment:
The pwm1 check should remain, in case the user is still running the old version of the driver.

And the preceeding comment should be adjusted accordingly.

Reply:
In that case, wouldn't it be better to not disregard the old version of the driver and only read the coolant temp and pump speed from it?

Reply:
You're right, that would be better.

Reply:
I've added a warning if it can't be read, similar to the [aquacomputer](https://github.com/liquidctl/liquidctl/blob/main/liquidctl/driver/aquacomputer.py#L375) driver.